concepticondata/conceptlists/DiezAlamo-2018-750.tsv

```diff
@@ -0,0 +1,751 @@
+DiezAlamo-2018-750-351	351	PURO	cigar			3.375	3.26923076923077	7.11538461538461	4.04166666666667	1.375	2.16666666666667	5.625		3.652		
+DiezAlamo-2018-750-352	352	CIGARRILLO	cigarette	CIGARETTE	969	2.91666666666667	3.76923076923077	7.76923076923077	5.04166666666667	1.33333333333333	2.45833333333333	6.75		6.299	5.832	
+DiezAlamo-2018-750-353	353	ALMEJA	clam			3.66666666666667	3.42307692307692	7.57692307692308	1.79166666666667	1.33333333333333	5.95833333333333	5.41666666666667				
+DiezAlamo-2018-750-354	354	ABRAZADERA	clamp			3	2.30769230769231	4.88461538461539	2.58333333333333	2.58333333333333	2.29166666666667	1.91666666666667				Flores (A flower)
+DiezAlamo-2018-750-355	355	CLARINETE	clarinet			4.95833333333333	3.46153846153846	6.96153846153846	2.29166666666667	5.08333333333333	2.79166666666667	1.75				Instrumentos musicales (A musical instrument)
+DiezAlamo-2018-750-356	356	RELOJ	clock	CLOCK	1323	4.16666666666667	5.73076923076923	7.69230769230769	1.91666666666667	2.95833333333333	2.20833333333333	1.66666666666667		6.518	6.043	Prendas de vestir (An article of clothing)
+DiezAlamo-2018-750-357	357	VESTIDOR	closet			4.20833333333333	1.61538461538462	1.61538461538462	1.54166666666667	2.04166666666667	2.625	3.16666666666667				Muebles (An article of furniture)
+DiezAlamo-2018-750-358	358	CUCARACHA	cockroach	COCKROACH	1575	4.58333333333333	6.73076923076923	6.88461538461539	2.41666666666667	1.45833333333333	1.33333333333333	1.83333333333333				Insectos (An insect)
+DiezAlamo-2018-750-359	359	COCO	coconut	COCONUT	970	4.33333333333333	2.69230769230769	7.46153846153846	3.29166666666667	2.08333333333333	6.95833333333333	4.375		5.548		Frutas (A fruit)
+DiezAlamo-2018-750-360	360	BACALAO	cod			3.375	6.19230769230769	6.38461538461539	1.70833333333333	1.5	6.25	5.625		6.039	6.534	Peces (A fish)
+DiezAlamo-2018-750-361	361	MONEDA	coin	COIN	1867	5.29166666666667	4.15384615384615	7.96153846153846	1.66666666666667	2.33333333333333	1.91666666666667	3.20833333333333		5.873	6.295	Dinero (A kind of money)
+DiezAlamo-2018-750-362	362	ESCURRIDOR	colander			2.83333333333333	2.76923076923077	7.38461538461539	1.45833333333333	1.91666666666667	2.125	1.95833333333333				Utensilios de cocina (A kitchen utensil)
+DiezAlamo-2018-750-363	363	CORCHO	cork			3.04166666666667	2	7.23076923076923	1.875	2.08333333333333	2.5	2.45833333333333				Partes de un edificio (A part of a building)
+DiezAlamo-2018-750-364	364	SACACORCHOS	corkscrew			3.33333333333333	3.26923076923077	7.76923076923077	4	2.33333333333333	2.16666666666667	1.75				Utensilios de cocina (A kitchen utensil)
+DiezAlamo-2018-750-365	365	MAÍZ	corn	MAIZE	506	5.91666666666667	2.26923076923077	7.65384615384615	1.45833333333333	1.70833333333333	6.875	4		6.406	6.156	
+DiezAlamo-2018-750-366	366	CHALET	cottage			3.83333333333333	1.26923076923077	1.15384615384615	1.58333333333333	2.95833333333
```

Comment:
Not POT, "maceta" is a "little mace/hammer", but can also just mean the handle of a tool/instrument.